aws/data_source_aws_lb_target_group_test.go

```diff
@@ -27,6 +27,7 @@ func TestAccDataSourceAWSALBTargetGroup_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("data.aws_lb_target_group.alb_tg_test_with_arn", "protocol", "HTTP"),
 					resource.TestCheckResourceAttrSet("data.aws_lb_target_group.alb_tg_test_with_arn", "vpc_id"),
 					resource.TestCheckResourceAttr("data.aws_lb_target_group.alb_tg_test_with_arn", "deregistration_delay", "300"),
+					resource.TestCheckResourceAttr("data.aws_lb_target_group.alb_tg_test_with_arn", "slow_start", "0"),
```

Comment:
The data source testing is failing because `d.Set("slow_start", ...)` is never called in the resource read function. This would normally be caught in the resource acceptance testing with a `TestCase` that calls `ImportStateVerify: true`.

```
=== RUN   TestAccDataSourceAWSALBTargetGroup_basic
--- FAIL: TestAccDataSourceAWSALBTargetGroup_basic (494.14s)
    testing.go:518: Step 0 error: Check failed: 2 error(s) occurred:
        
        * Check 9/39 error: data.aws_lb_target_group.alb_tg_test_with_arn: Attribute 'slow_start' not found
        * Check 28/39 error: data.aws_lb_target_group.alb_tg_test_with_name: Attribute 'slow_start' not found
```